### PR TITLE
Describe boot.md file

### PR DESCRIPTION
**Pull request by Gamma-Zero**

### CODE REVIEW

File: boot.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# [BootManager](../src/vmm/src/boot.rs)
```
Comment (Krishnanshu1907) [inline boot.md:1]: Testing

Comment (rishi1001) [inline boot.md:1]: Testing

File: boot.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+There are 3 terms related to memory being used here:
+* `Guest Memory`: The memory available to the VM.
+* `High Memory`: The guest memory is divided into 2 parts: Low Memory and High Memory. The kernel uses the Low Memory and the user uses High Memory. PTEs for High Memory are not maintained permanently, but rather as they are needed.
+* `Memory Mapped for I/O`: This is the memory mapped for I/O devices.
```
Comment (aayushgoyal443) [inline boot.md:6]: Testing

File: boot.md (hunk 1 above)
Comment (Prakhar314) [inline boot.md:1]: Testing

Comment (gbnm2001) [inline boot.md:1]: Testing

**Comment by codenet** [comment]
Duplicate of https://github.com/codenet/vmm-reference/pull/4

Add changes unique to your PR directly in the other PR.

**Pull request closed without merging**